File: Linguaggio_Alto_Livello/JavaScript/Medium/count_vowels.js
const text = "Imparare JavaScript e' divertente";
const vowels = new Set(["a", "e", "i", "o", "u"]);
let count = 0;

for (const char of text.toLowerCase()) {
  if (vowels.has(char)) {
    count += 1;
  }
}

console.log(`Testo: ${text}`);
console.log(`Numero di vocali: ${count}`);

File: Linguaggio_Alto_Livello/Java/Medium/CountVowels.java
public class CountVowels {
    public static void main(String[] args) {
        String text = "Imparare Java e' utile";
        int count = 0;
        for (char ch : text.toLowerCase().toCharArray()) {
            if ("aeiou".indexOf(ch) >= 0) {
                count++;
            }
        }
        System.out.println("Testo: " + text);
        System.out.println("Numero di vocali: " + count);
    }
}
